app/governance/[id]/proposalModal.module.scss

```diff
@@ -0,0 +1,168 @@
+    }
+    .proposalCardContainer1{
+        display: flex;
+        flex-direction: column !important;
+        justify-content: space-between;
+        //background-color: green;
+        // width: 50% !important;
+        //height: 100%;
+        width: 50%;
+        height: 80%;
+
+        
+    }
+    .proposalCardContainer2{
+        display: flex;
+        flex-direction: column !important;
+        justify-content: space-between;
+        //background-color: blue;
+        height: 80%;
+        width: 50%;
+        background-color: var(--card-sub-surface-color, #dfdfdf);
+        box-shadow: 3px 3px 0px rgba(41, 40, 40, 0.2);
+        
+        // background-color: red;
+    }
+    .proposalInfoBox{
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        background-color: var(--card-sub-surface-color, #dfdfdf);
+        margin: 0px 20px 0px 0px;
+        height: 40%;
+        box-shadow: 3px 3px 0px rgba(41, 40, 40, 0.2);
+        padding: 10px;
+        //align-items: center;
+        //justify-content: space-between;
+        //margin: 10% 10% 10% 10%;
+        .proposalInfo{
+        
+            display: flex;
+            flex-direction: row;
+            justify-content: space-between;
+            //overflow: ;
+        }
+    }
+    .proposalInfoBoxVoting{
+        background-color: var(--card-sub-surface-color, #dfdfdf);
+        //height: 500px;
+        //height: 60%;
+        padding: 10px;
+
+        .proposalInfoVoting{
+        
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            flex-wrap: nowrap;
+            border: 1px #B3B3B3 solid;
+            border-radius: 5px ;
+            padding: 5px;
+            //border-radius: 2px;
+            height: 100%;
+            width: 45%;
+            
+        }
+        .proposalInfoRow1{
+            display: flex;
+            flex-direction: row !important;
+            justify-content: space-between;
+            height: 50%;
+            margin: 10px;
+            padding: 10px;
+        }
+        .votingInfoRow1{
+            display: flex;
+            flex-direction: row;
+            justify-content: space-between;
+            .circle {
+                width: 7px; /* Small circle size */
+                height: 7px; /* Small circle size */
+                //background-color: red; /* Red circle */
+                border-radius: 50%; /* Make it round */
+                //margin-top: 20%;
+                //position: absolute; /* Position it inside the box */
+              }
+        }
+        .votingInfoRow2{
+            display: flex;
+            flex-direction: row;
+            justify-content: right;
+        }
+
+    }
+    .VotingButton{
+        //padding-bottom: 20px;
+        //width: 100%;
+        margin: auto;
+        align-items: center;
+
+        
+    }
+
+    .VotingButton .tooltip {
```

Comment:
we already have a tooltip component please use that to stay consistent with site's design